contracts/anchor/Anchor.sol

```diff
@@ -64,12 +64,12 @@ contract Anchor is AnchorI, ConsensusModule, CircularBufferUint {
         uint256 _blockHeight,
         bytes32 _stateRoot,
         uint256 _maxStateRoots,
-        address _consensus
+        ConsensusI _consensus
```

Comment:
Just curious why `ConsensusI _consensus` instead of `address _consensus`?

Reply:
We know the type of **consensus**. So no need to have an generic type **address**.

Reply:
similar to reputation.sol, neither anchor or reputation currently call a function on `consensus`, so there is a strong argument to use `address`; however we currently have in reputation used the interface, so we can consistently here use the interface